scripts/sparkclr-submit.cmd

```diff
@@ -5,10 +5,10 @@ if "%JAVA_HOME%" == "" goto :javahomeerror
 if "%SPARKCLR_HOME%" == "" goto :sparkclrhomeerror
 
 if "%SPARK_CONF_DIR%" == "" (
-	SET SPARK_CONF_DIR=%SPARK_HOME%\conf
+	SET SPARK_CONF_DIR="%SPARK_HOME%\conf"
 )
 
-call %SPARK_HOME%\bin\load-spark-env.cmd
+call "%SPARK_HOME%\bin\load-spark-env.cmd"
```

Comment:
I think this quoting is also required at Line 55 around the spark-submit.cmd call?